docs/development/sync-protocol.md

```diff
@@ -236,6 +236,9 @@ sequenceDiagram
 | signing_id | NO | string | Signing ID of the binary that was executed | "EQHXZ8M8AV:com.google.Chrome" |
 | team_id | NO | string | Team ID of the binary that was executed | "EQHXZ8M8AV" |
 | cdhash | NO | string | CDHash of the binary that was executed | "dbe8c39801f93e05fc7bc53a02af5b4d3cfc670a" |
+| entitlementInfo | NO | Entitlement Info Object (see below) | Entitlement information about the target executable | see below |
+| csFlags | NO | int32 | The raw codesigning flags set for the execution |
+| signingStatus | NO | The signing status of the executing binary | **Must be one of the examples** | "SIGNING_STATUS_UNSPECIFIED", "SIGNING_STATUS_UNSIGNED", "SIGNING_STATUS_ADHOC", "SIGNING_STATUS_DEVELOPMENT", "SIGNING_STATUS_PRODUCTION" |
```

Comment:
Is this the right casing? I would've thought it was words separated by underscores. If not, we should probably fix to match the others.

Reply:
It is the right casing per what Santa does today as of 2025.1

Reply:
For 2025.2 we should fix it to be consistent.